logbook/logbook_test.go

```diff
@@ -73,7 +73,7 @@ func Example() {
 
 	// create a log record of the version of a dataset. In practice this'll be
 	// part of the overall save routine that created the above ds variable
-	if err := book.WriteVersionSave(ctx, ds); err != nil {
+	if _, err := book.WriteVersionSave(ctx, ds); err != nil {
```

Comment:
The fact that we're ignoring this response so often (even in tests) to me is an indication that we should have some sort of internal wiring where _any_ call to WriteVersionSave is publishing an `Event` to a channel that others can subscribe to, instead of returning actions.